common/scripted_effects/wc_exp1_historical_artifacts_creation_effect.txt

```diff
@@ -0,0 +1,27 @@
+﻿create_artifact_bloodhoof_runespear_effect = {
```

Comment:
Why there are 2 effects that add the runespear?

Reply:
Because when I started the branch, I peeked at your frostmourne branch, where you used the wc_exp1_ file. Now when I continued, I checked your terenas' crown branch, because that works from the start and not via event. There you used the wc_historical_artifacts... file, so i took that one (for now) as well... Especially since I didn't know which file would/should work better...